android/app/src/main/java/com/zulipmobile/MainActivity.kt

```diff
@@ -1,8 +1,22 @@
 package com.zulipmobile
 
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
 import android.os.Bundle
+import android.util.Log
 import android.webkit.WebView
+import androidx.annotation.Nullable
 import com.facebook.react.ReactActivity
+import com.facebook.react.ReactApplication
+import com.facebook.react.bridge.Arguments
+import com.facebook.react.bridge.ReactContext
+import com.facebook.react.bridge.WritableMap
+import com.zulipmobile.notifications.*
+import com.zulipmobile.sharing.SharingModule
+
+const val TAG = "MainActivityShare"
```

Comment:
Should just be "MainActivity", because it's global to this file so it's the natural thing that other logging calls in this file will use. Or leave it specific to sharing but name it something specific like `SHARE_TAG`.